### PR TITLE
Updates for multiple libraries in Sketch with 3.6

### PR DESCRIPTION
**Pull request by StefanIvanov**
Updates are necessary for the fact that in Sketch we split the library in… 4 variants, one for each design language we support

### CODE REVIEW

**Comment by StefanIvanov** [comment]
Thanks @randriova now it is up to us to review, test and merge this for the release of 3.6.0 @zdrawku could someone on your team help with the tests?

File: en/components/components-overview.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The **Indigo.Design System** empowers your design workflow with 60+ UI Component
...
 
-The Sketch library has a `🧩 Components` page containing all Components and symbols related to them. When selecting a Symbol instance, through the Override section in the right panel, one can change the content inside e.g. enabled/disabled states, light/dark variants, icons, colors, etc. Below is a small part of showing some of the Components and symbols such as the different types of [Buttons](button.md), [Button Group](button-group.md) and form elements like [Checkbox](checkbox.md), [Switch](switch.md), and [Radio Group](radio-group.md).
+The Sketch libraries have a `🧩 Components` page containing all Components and symbols related to them. When selecting a Symbol instance, through the Override section in the right panel, one can change the content inside e.g. enabled/disabled states, light/dark variants, icons, colors, etc. Below is a small part of showing some of the Components and symbols such as the different types of [Buttons](button.md), [Button Group](button-group.md) and form elements like [Checkbox](checkbox.md), [Switch](switch.md), and [Radio Group](radio-group.md).
```
Comment (YoannaIvanova) [inline en/components/components-overview.md:14]:
```suggestion
The Sketch libraries have a `🧩 Components` page containing all Components and symbols related to them. When selecting a Symbol instance, through the Override section in the right panel, one can change the content inside e.g. enabled/disabled states, icons, colors, etc. Below is a small part of showing some of the Components and symbols such as the different types of [Buttons](button.md), [Button Group](button-group.md) and form elements like [Checkbox](checkbox.md), [Switch](switch.md), and [Radio Group](radio-group.md).
```

File: en/components/components-overview.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ The **Indigo.Design System** empowers your design workflow with 60+ UI Component
...
 
 <img class="responsive-img" src="../images/components-page.png" />
```
Comment (YoannaIvanova) [inline en/components/components-overview.md:16]: We need to update the image. There are light/dark variants and overrides that we removed.

File: en/components/components-overview.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ In the Adobe XD library you can find the `🧩 Components` in the middle column.
 <img class="responsive-img" src="../images/components-page-xd.png" />
```
Comment (YoannaIvanova) [inline en/components/components-overview.md:20]: We also need to update this image.

File: en/components/components-overview.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ In the Adobe XD library you can find the `🧩 Components` in the middle column.
...
 
 > [!WARNING]
-> Detaching a component from the Sketch library makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers, containing metadata descriptions of the state, template and other properties of the component.
+> Detaching a component from the library in Sketch makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers, containing metadata descriptions of the state, template and other properties of the component.
```
Comment (YoannaIvanova) [inline en/components/components-overview.md:23]:
```suggestion
> Detaching a component from the libraries in Sketch makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers, containing metadata descriptions of the state, template and other properties of the component.
```

File: en/getting-started.md [hunk 5]
```diff
@@ -24,9 +24,11 @@ Indigo.Design offers three key capabilities to support the design to development
...
+The design system is a set of libraries for Sketch and Adobe XD, containing various assets, styles, components, and patterns. Тhe UI components allow you to create intricate user interfaces providing you with the flexibility to use various states and templates. To customize their appearance you may leverage the pre-made palettes, typographies, shadows, icons, and illustrations. And finally, with the provided collection of patterns, such as product details layouts, various forms, and pricing tables among others, you are able to streamline app design assuring the application of best practices in UI and interaction design.
 
-Learn more:
+Indigo.Design for Sketch now offers not one but four library files to its users: one for each of the design languages we support - Material, Bootstrap, Fluent UI and our very own Indigo. All four libraries support seamless switching between light and dark mode via the [Indigo.Design plugin](sync-themes-plugin.md) and work with any of the predefined or custom color palettes it provides. With this improvement Sketch users are able to enjoy 100% native look and feel of their components and patterns for the design language of their choice.
```
Comment (YoannaIvanova) [inline en/getting-started.md:29]:
```suggestion
Indigo.Design for Sketch now offers not one but four library files to its users: one for each of the design languages we support - Material, Bootstrap, Fluent UI, and our very own Indigo. All four libraries support seamless switching between light and dark mode via the [Indigo.Design plugin](sync-themes-plugin.md) and work with any of the predefined or custom color palettes it provides. With this improvement, Sketch users are able to enjoy 100% native look and feel of their components and patterns for the design language of their choice.
```

File: en/components/components-overview.md [hunk 6]
```diff
@@ -6,21 +6,25 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
...
+## Adobe XD
 
+In the Adobe XD library you can find the `🧩 Components` in the middle column. Here, the various states of a component can be switched from the right panel. Also, we have used the `Stack` and `Padding` to align and distribute elements vertically or horizontally and change their position. When you want to swap a nested component with another one, you can simply drag it from the `Libraries` panel and drop it on top of the current one to replace it. You will also notice that some components, such as the Grid, for example, use a `Repeat Grid` to speed configuration up.
+> Detaching a component from the libraries in Sketch makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers, containing metadata descriptions of the state, template and other properties of the component.
```
Comment (YoannaIvanova) [inline en/components/components-overview.md:23]: Paragraphs on row 23 and row 27 are duplicated. And the text in row 23 doesn't correspond to Adobe XD (see the image below).

Comment (StefanIvanov) [reply en/components/components-overview.md:23]: This duplication seems to be a merge issue from the downtime of github yesterday, I am pushing a fix. About the paragraph content I am making a small edit but the idea is for it to give the overall idea for what XD mechanisms we provide to the people familiar with XD already and the image is more like an add-on with some components.

**Comment by randriova** [comment]
@StefanIvanov Please let me know once you've finished the EN changes and I will update JP as well. Thanks :)

**Comment by StefanIvanov** [comment]
@randriova we had some issues with Github yesterday but now they should be all set. You can finish the translations, I expect that @YoannaIvanova will approve the changes and we will be all set to merge this today.

**Review by YoannaIvanova — DISMISSED** [review]

**Review by YoannaIvanova — APPROVED** [review]

**Pull request merged**